src/scss/_color-use-cases.scss

```diff
@@ -8,16 +8,31 @@
 @include oColorsSetColor('black-crimson-12.5', oColorsMix('black', 'crimson', 12.5)); //text
 @include oColorsSetColor('paper-crimson-90', oColorsMix('paper', 'crimson', 90)); //background
 
-@include oColorsSetUseCase(o-message-alert-error, text, 'black-crimson-12.5');
-@include oColorsSetUseCase(o-message-alert-error, background, 'paper-crimson-90');
+@include oColorsSetUseCase(o-message-error, text, 'black-crimson-12.5');
+@include oColorsSetUseCase(o-message-error, background, 'paper-crimson-90');
 
 /// Neutral status styles
-@include oColorsSetUseCase(o-message-alert-neutral, background, 'white-60');
-@include oColorsSetUseCase(o-message-alert-neutral, text, 'black');
+@include oColorsSetUseCase(o-message-neutral, text, 'black');
+@include oColorsSetUseCase(o-message-neutral, background, 'white-60');
 
 /// Success status styles
 @include oColorsSetColor('black-jade-43', oColorsMix('black', 'jade', 43)); //text
 @include oColorsSetColor('paper-jade-80', oColorsMix('paper', 'jade', 80)); //background
 
-@include oColorsSetUseCase(o-message-alert-success, text, 'black-jade-43');
-@include oColorsSetUseCase(o-message-alert-success, background, 'paper-jade-80');
+@include oColorsSetUseCase(o-message-success, text, 'black-jade-43');
+@include oColorsSetUseCase(o-message-success, background, 'paper-jade-80');
+
+/// Status Colors for Notice Messages
+/// Inform status styles
+@include oColorsSetUseCase(o-message-inform, text, 'black');
```

Comment:
We've been deprecating custom color usecases as part of branding. Maybe we should brand o-messages and remove the usecases before this breaking? Not a big issue if you'd rather release separately.

Reply:
I think I'd rather do that separately, and just get this out the door for now

Reply:
👍